cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -175,6 +175,7 @@ visualization_msgs::MarkerArray MapBuilderBridge::GetTrajectoryNodeList() {
        ++trajectory_id) {
     const auto& single_trajectory_nodes = all_trajectory_nodes[trajectory_id];
     visualization_msgs::Marker marker;
+    marker.ns = "Trajectory " + std::to_string(trajectory_id);
     marker.id = marker_id++;
```

Comment:
Yes, but I was arguing for the opposite: Every trajectory now has its own `ns`, so can have a markers with `id` 0, 1, 2, etc. Only the pair of `ns` and `id` has to be unique, see https://github.com/ros-visualization/rviz/blob/kinetic-devel/src/rviz/default_plugin/marker_display.h#L62